easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+
+homepage = 'http://www.popgen.dk/software/index.php/NgsAdmix'
+description = """This page contains information about the program called 
+NGSadmix, which is a tool for finding admixture proportions 
+from NGS data, based on genotype likelihoods."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+# This is a bit of a hack as there is no easyblock to simply download a cpp file and compile it.
+# So we download the file from the source and copy it to the build directory. 
+
+sources = [{
+    'source_urls': ['http://popgen.dk/software/download/NGSadmix/'],
+    'filename': 'ngsadmix32.cpp',
+    'download_filename': 'ngsadmix32.cpp',
+    'extract_cmd': 'cp %s %(builddir)s',
+}]
+
+checksums = ['a5623b71d20114708cfd7ccc5f8c6ce38ff871d1b79af85bc89d38775f2d477e']
+
+build_str = "g++ ngsadmix32.cpp -O3 -lpthread -lz -o NGSadmix"
```

Comment:
Please refrain from hardcoding the compiler command and optimization options, use `$CXX` and `$CXXFLAGS` instead (which are defined based on the toolchain being used), and also use `%(version)s` rather than hardcoding `32` as version.
I also see no need to use a local variable `build_str`?

```python
cmds_map = [('', "${CXX} ngsadmix%(version)s.cpp ${CXXFLAGS} -lpthread -lz -o NGSadmix")]

Reply:
I think that came from a different script file. I changed it. Thanks.